components/director/internal/domain/document/converter_test.go

```diff
@@ -1 +1,144 @@
-package document
+package document_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/internal/domain/document"
+	"github.com/kyma-incubator/compass/components/director/internal/domain/document/automock"
+	"github.com/kyma-incubator/compass/components/director/internal/graphql"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConverter_ToGraphQL(t *testing.T) {
+	// given
+	testCases := []struct {
+		Name     string
+		Input    *model.Document
+		Expected *graphql.Document
+	}{
+		{
+			Name:     "All properties given",
+			Input:    fixModelDocument("1", "foo"),
+			Expected: fixGQLDocument("foo"),
+		},
+		{
+			Name:     "Empty",
+			Input:    &model.Document{},
+			Expected: &graphql.Document{},
+		},
+		{
+			Name:     "Nil",
+			Input:    nil,
+			Expected: nil,
+		},
+	}
+
+	for i, testCase := range testCases {
+		t.Run(fmt.Sprintf("%d: %s", i, testCase.Name), func(t *testing.T) {
```

Comment:
what do you add "i" to the test name?
```
FAIL: TestConverter_ToGraphQL/0:_All_properties_given (0.00s)
```

Reply:
I was adding it to stay consistent with how previous tests (runtime and a part of application) were written.